src/main/java/com/paneedah/weaponlib/ServerEventHandler.java

```diff
@@ -172,7 +175,7 @@ protected void onPlayerStartedTracking(PlayerEvent.StartTracking e) {
                     new EntityInventorySyncMessage(e.getTarget(), 
                             CompatibleCustomPlayerInventoryCapability.getInventory((EntityLivingBase) e.getTarget()), false), 
                             (EntityPlayerMP) e.getEntityPlayer());
-            System.out.println("Player " + e.getEntityPlayer() + " started tracking "  + e.getTarget());
+            //System.out.println("Player " + e.getEntityPlayer() + " started tracking "  + e.getTarget());
```

Comment:
It is preferable to just delete this, anyway thanks to GitHub if we ever need it we can just see it in the history.